backend/src/api/routes/index.ts

```diff
@@ -1,8 +1,10 @@
 import { Express } from 'express';
 import userRoute from './userRoute';
+import workspaceRoute from './workspaceRoute';
 
 const routes = (app: Express) => {
   app.use('/api/users', userRoute);
+  app.use('/api', workspaceRoute);
```

Comment:
```suggestion
  app.use('/api/workspaces', workspaceRoute);
```

there is an example above